process.py

```diff
@@ -386,9 +387,15 @@ def get_provider_code(directory_name=None):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--path', type=str, default="static/data/lms")
+    specific_folder = raw_input('Do you want to load a specific folder? type y to load a specific folder, type n to load all the data\n')
+    if specific_folder == 'y' or specific_folder == 'Y':
+        dirname = tkFileDialog.askdirectory(initialdir=os.path.abspath("static/data/lms"),  title='Please select a directory')
+        parser.add_argument('--delete_all', dest='delete_all', action='store_true', default=False)
+    else:
+        dirname = "static/data/lms"
+        parser.add_argument('--delete_all', dest='delete_all', action='store_true', default=True)
+    parser.add_argument('--path', type=str, default=dirname)
```

Comment:
@danielhers I haven't erased it. I just moved it here so the directory name will be either static/data/lms or the chosen directory.